processes/consumer/process.go

```diff
@@ -35,7 +35,7 @@ func processMessage(ctx context.Context, cfg config.Config, inMemDB *models.Data
 	}
 	st := time.Now()
 	defer func() {
```

Comment:
Do we need this `func(){}` here?

Reply:
Are you just asking about the func wrapper?

As opposed to just doing:
```go

defer metricsClient.Timing("process.message", time.Since(st), tags)
```

Reply:
Yep.

Reply:
Turns out you do need it or `time.Since(st)` evaluates to ~0.